setup.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python2
```

Comment:
Why you removed `env`?
Change it to `/usr/bin/env python2`

Reply:
Done